src/agent/README.md

```diff
@@ -113,3 +113,43 @@ agent control tool, rather than the Kata runtime.
 
 For further details, see the
 [agent control tool documentation](../tools/agent-ctl/README.md#run-the-tool-and-the-agent-in-the-same-environment).
+
+## Agent options
+
+The kata agent has the ability to configure agent options in guest kernel command line at runtime. Currently, the following agent options can be configured: 
+
+| Option | Description |  Value | Type | Default |
+|-|-|:-:|:-:|:-:|
+| `Debug console flag` | Allow to connect guest OS running inside hypervisor through "kata-runtime exec <sandbox-id>" command | `agent.debug_console` | / | / |
```

Comment:
- Nit: I don't think you need to have the first column in back ticks
- Could you change the last two columns in this row to "boolean" and `false`.